ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionsReader.scala

```diff
@@ -390,4 +384,21 @@ private[dao] final class TransactionsReader(
     }
     mat
   }
+
+  override def getContractStateEvents(startExclusive: (Offset, Long), endInclusive: (Offset, Long))(
+      implicit loggingContext: LoggingContext
+  ): Source[((Offset, Long), ContractStateEvent), NotUsed] =
+    throw new UnsupportedOperationException(
+      "The operation is not supported in the current version."
```

Comment:
The reason for not being supported is that it is not yet implemented isn't it?

The message currently just repeats the name of the exception. Consider a message like "getContractStateEvents: not yet implemented."

Reply:
Good point, done